### PR TITLE
V1.4 dg improvement

### PR DESCRIPTION
**Pull request by hernpiblo**
Improve the overall formatting of the DG
 Add final points to hernpiblo PPP

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -467,7 +439,7 @@ During the SetTablesCommand execution:
 4. There is no add/remove/edit table or table list
     * Again, this is because reservations are linked to the tables themselves and if the user alters the list of tables, the reservations will not be in sync.
 
-#### Future Implementations
+**Future Implementations:**
```
Comment (clementkfj) [inline docs/DeveloperGuide.md:442]: Actually do we need this part?

Comment (hernpiblo) [reply docs/DeveloperGuide.md]: MB. It was supposed to be called alternative considered as per the textbook instructions. Thanks for spotting that !

Comment (clementkfj) [reply docs/DeveloperGuide.md]: Hey no problem!

**Review by clementkfj — APPROVED** [review]
LGTM!

**Pull request merged**